atd-vze/.env

```diff
@@ -1,5 +1,5 @@
 REACT_APP_HASURA_ENDPOINT="https://vzd-staging.austinmobility.io/v1/graphql"
-# REACT_APP_HASURA_ENDPOINT="http://localhost:8080/v1/graphql"
+# REACT_APP_HASURA_ENDPOINT="http://localhost:8084/v1/graphql"
```

Comment:
Should switch this to default to the local endpoint? I've been using local more and more by default but not sure about everyone else.

Reply:
but in the meantime, sure, change it to default to local endpoint!

Reply:
defaulting to local sounds good, i have found myself using it more often as well

Reply:
thanks y'all! I'll go ahead and switch the commented lines here.